src/main/scala/mesosphere/marathon/core/task/bus/MesosTaskStatus.scala

```diff
@@ -36,7 +38,7 @@ object MesosTaskStatus {
   object TemporarilyUnreachable {
     def isUnreachable(task: MarathonTask): Boolean = isUnreachable(task.getStatus)
     def isUnreachable(taskStatus: TaskStatus): Boolean = {
-      taskStatus.getState == TASK_LOST && MightComeBack(taskStatus.getReason)
+      taskStatus.getState == TASK_LOST && MightComeBack(taskStatus.getReason) && !wontComeBack(taskStatus)
```

Comment:
I'd have preferred a `def mightComeBack` in contrast to the double negation, but I can live with it in releases/0.15 👍